modules/nested-elements/assets/js/editor/nested-repeater/views/widget-repeater.js

```diff
@@ -0,0 +1,90 @@
+const WidgetView = elementor.modules.elements.views.Widget;
+
+/**
+ * @extends {BaseElementView}
+ */
+export class WidgetRepeater extends elementor.modules.elements.views.BaseElement {
+	initialize() {
+		this.config = elementor.widgetsCache[ this.options.model.attributes.widgetType ];
+
+		this.once( 'container:created', this.onContainerCreatedOnce.bind( this ) );
+
+		WidgetView.prototype.initialize.apply( this, arguments );
+	}
+
+	events() {
+		const events = super.events();
+
+		events.click = ( e ) => {
+			const $parentsUntil = jQuery( e.target ).parentsUntil( '.elementor-widget-container' );
+
+			// Clicks higher then 3, are nested, should not be handled by current.
+			if ( 3 <= $parentsUntil.length ) {
+				return true;
+			}
+
+			e.stopPropagation();
+
+			$e.run( 'panel/editor/open', {
+				model: this.options.model,
+				view: this,
+			} );
+		};
+
+		return events;
+	}
+
+	getChildViewContainer( containerView, childView ) {
+		if ( this.config.children_placeholder_selector ) {
+			return containerView.$el.find( this.config.children_placeholder_selector );
+		}
+
+		return super.getChildViewContainer( containerView, childView );
+	}
+
+	getChildType() {
+		return [ 'container' ];
+	}
+
+	onRender() {
+		super.onRender();
+
+		const editModel = this.getEditModel(),
+			skinType = editModel.getSetting( '_skin' ) || 'default';
+
+		// To support handlers.
+		this.$el
+			.attr( 'data-widget_type', editModel.get( 'widgetType' ) + '.' + skinType )
+			.removeClass( 'elementor-widget-empty' )
+			.children( '.elementor-widget-empty-icon' )
+			.remove();
+	}
+
+	onContainerCreatedOnce() {
+		const { defaultChildren = [] } = this.container.model;
+
+		if ( defaultChildren.length ) {
+			defaultChildren.forEach( ( elementModel ) => this.addElement( elementModel, { internal: true } ) );
+		}
+	}
+
+	addElement( data, options = {} ) {
+		const container = this.container;
+
+		options.onAfterAdd = ( model ) => {
+			container.view.trigger( 'nested-repeater:add-element:after', { model, options } );
```

Comment:
Explain

Reply:
https://github.com/elementor/elementor/pull/16252/commits/5989c02dc5b21394867f70cb78e1679ca3740e92

Reply:
Why can't it run like other hooks without JS events?

Reply:
$e.hooks doesn't know when view finish render, i think we can do `setTimeout` on render and then trigger that render is done, what do you think?

Reply:
Try the `setTimeout`